inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ if(COMMAND ie_add_vs_version_file)
 endif()
 
 if(TARGET offline_transformations)
-    list(APPEND InferenceEngine_LIBRARIES offline_transformations)
+    list(APPEND InferenceEngine_LIBRARIES offline_transformations commonTestUtils)
 else()
-    list(APPEND InferenceEngine_LIBRARIES IE::offline_transformations)
+    list(APPEND InferenceEngine_LIBRARIES IE::offline_transformations IE::commonTestUtils)
```

Comment:
I supposed that python offline API is release component, while `commonTestUtils` and other test utils - does not. 

Do we need it only for usage in tests?

Reply:
Moved to a separate python module.